redis/tests/test_json.rs

```diff
@@ -340,19 +364,19 @@ fn test_json_numincrby() {
 
     assert_eq!(set_initial, Ok(true));
 
-    let json_numincrby_a: RedisResult<String> = con.json_numincrby(TEST_KEY, "$.a", 2);
+    let json_numincrby_a: RedisResult<String> = con.json_num_incr_by(TEST_KEY, "$.a", 2);
```

Comment:
Wait what? :joy:

Reply:
shiiii
probably should've read all the code I replaced lmao
will fix in bug/existing tests PR
ngl I wrote this on 1 hour of sleep